packages/react-native-client/android/src/main/java/io/fishjam/reactnative/VideoPreviewView.kt

```diff
@@ -8,16 +8,20 @@ import expo.modules.kotlin.AppContext
 class VideoPreviewView(
   context: Context,
   appContext: AppContext
-) : VideoView(context, appContext) {
+) : VideoView(context, appContext),
+  RNFishjamClient.OnLocalCameraTrackChangedListener {
   private var localVideoTrack: LocalVideoTrack? = null
 
-  private fun initialize() {
+  private fun trySetLocalCameraTrack() {
```

Comment:
Maybe `trySettingLocalCameraTrack` would be easier to read name?

Reply:
Not sure it is easier, imho `trySetLocalCameraTrack` is more concise and aligns with common naming conventions in programming.